examples/opentracing/requirements.txt

```diff
@@ -0,0 +1,4 @@
+opentelemetry-api
+opentelemetry-opentracing-shim
+redis
+redis_opentracing
```

Comment:
Are `opentelemetry-sdk` and `opentelemetry-ext-jaeger` missing?

Reply:
Fixed!